src/fragments/lib/logging/common/remote_configuration/remote_configuration.mdx

```diff
@@ -0,0 +1,230 @@
+You can remotely configure the Amplify Logger, enabling you to make changes to your logging levels or user allow list in your deployed applications.
+
+<Callout>
+The logging configurations you set remotely will overwrite the local log level and persist for future app sessions.
+</Callout>
+
+## Set up remote configuration backend resources
+
+<Callout>
+
+You can use Amplify CLI generate a CDK template and [add custom resources](https://docs.amplify.aws/cli/custom/cdk/).  
```

Comment:
You can use the Amplify CLI to [add custom resources](https://docs.amplify.aws/cli/custom/cdk/).

Reply:
Text updated `You can use the Amplify CLI to add custom CDK resources.` This matches with the previous callout in the Set up Logging page.